plugins/heindekort/extract.js

```diff
@@ -0,0 +1,5 @@
+function(page) {
+    var regex = /<figure\sitemscope="itemscope"\sitemtype="http:\/\/schema.org\/ImageObject"\sitemprop="image"\sclass="fjs-gallery-item"\sdata-hires="(https:\/\/[^"]*)"/;
```

Comment:
This sounds quite fragile as in if any of those many texts change, then the comic will be broken.
Are these really the minimal texts you need to identify and extract the strip?

Reply:
It was strange to see that the page was changed as soon as I uploaded this code. I will look at everything again, including the points you made about the cover and example.
Cheers, Rob